src/mavsdk_server/src/plugins/camera_server/camera_server_service_impl.h

```diff
@@ -169,6 +261,11 @@ class CameraServerServiceImpl final : public rpc::camera_server::CameraServerSer
 
         obj.definition_file_uri = information.definition_file_uri();
 
+        for (const auto& elem : information.camera_cap_flags()) {
+            obj.camera_cap_flags.push_back(translateFromRpcCameraCapFlags(
+                static_cast<mavsdk::rpc::camera_server::Information::CameraCapFlags>(elem)));
```

Comment:
the auto generate code is static_cast<mavsdk::rpc::camera_server::CameraCapFlags> missing Information domain. I just manual add **Information** in auto generate code.